nvflare/fuel/hci/conn.py

```diff
@@ -27,16 +28,20 @@
 
 LINE_END = "\x03"  # Indicates the end of the line (end of text)
 ALL_END = "\x04"  # Marks the end of a complete transmission (End of Transmission)
-
-
 MAX_MSG_SIZE = 1024
+MAX_DATA_SIZE = 512*1024*1024
```

Comment:
As pointed out by the unit test (https://github.com/NVIDIA/NVFlare/runs/5517455146?check_suite_focus=true), it suggest changing this line to

`512 * 1024 * 1024`